rust/src/treefile.rs

```diff
@@ -672,30 +667,24 @@ impl Treefile {
         self.parsed.releasever.as_deref().unwrap_or_default()
     }
 
-    pub(crate) fn get_rpmdb(&self) -> String {
-        let s: &str = match self.parsed.rpmdb.as_ref().unwrap_or(&DEFAULT_RPMDB_BACKEND) {
-            RpmdbBackend::Bdb => "bdb",
-            RpmdbBackend::Sqlite => "sqlite",
-            RpmdbBackend::Ndb => "ndb",
-        };
-        s.to_string()
-    }
-
     /// Error out if the backend is ndb, which Fedora doesn't support.
     /// xref https://bugzilla.redhat.com/show_bug.cgi?id=1938928
     pub(crate) fn validate_rpmdb(&self) -> CxxResult<()> {
         match self.parsed.rpmdb.as_ref() {
-            None | Some(&RpmdbBackend::Sqlite) | Some(&RpmdbBackend::Bdb) => Ok(()),
+            None
+            | Some(&RpmdbBackend::Target)
+            | Some(&RpmdbBackend::Sqlite)
+            | Some(&RpmdbBackend::Bdb) => Ok(()),
```

Comment:
Shouldn't `RpmdbBackend::Host` be in this list?

Reply:
Eh, no real value in keeping this.  I don't think anyone is specifying `ndb` today, let's just treat it as `target` too.